mage/integrationTests/lib.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 Adobe
+// All Rights Reserved.
+//
+// NOTICE: Adobe permits you to use, modify, and distribute this file in
+// accordance with the terms of the Adobe license agreement accompanying
+// it. If you have received this file from a source other than Adobe,
+// then your use, modification, or distribution of it requires the prior
+// written permission of Adobe.
+
+package integrationTests
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	harness "github.com/kudobuilder/kuttl/pkg/apis/testharness/v1beta1"
+	"github.com/kudobuilder/kuttl/pkg/test"
+	testutils "github.com/kudobuilder/kuttl/pkg/test/utils"
+	// mage:import utils
+)
+
+// RunIntegrationTests Runs Integration Tests using KUTTL
```

Comment:
mage will pickup target name from func definition, we don't need to repeat it.
```suggestion
//Runs Integration Tests using KUTTL
```

Reply:
Removed